packages/swapper/src/api.ts

```diff
@@ -20,6 +22,12 @@ type ChainSpecificQuoteFeeData<T extends ChainId> = ChainSpecific<
       approvalFee?: string
       totalFee?: string
     }
+    'eip155:43114': {
```

Comment:
Can we use the `KnownChainIds` in place of hardcode strings?

Reply:
Actioned in https://github.com/shapeshift/lib/pull/850/commits/6c496b3b156797f381fafe0f13a20d7c418fca6c - it required some propagation.